ALUM/ALUM/Views/PreSessionForm/PreSessionConfirmationScreen.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  PreSessionConfirmationScreen.swift
+//  ALUM
+//
+//  Created by Neelam Gurnani on 3/9/23.
+//
+// Notes:
+// The button titled "XXX" currently leads back to the login page, check where it should actually be leading back to.
+
+import SwiftUI
+
+struct PreSessionConfirmationScreen: View {
+    @ObservedObject var viewModel: QuestionViewModel
+    @Environment(\.dismiss) var dismiss
```

Comment:
In the future, try to include a screenshot of any frontend pages/components that you created or updated in the PR. That's what the "Confirmation of Change" section in the template is for